content/docs/ref/index.md

```diff
@@ -1,5 +1,18 @@
 # Command Reference
 
+CML provides a number of commands to help package the outputs of ML workflows
+into a CML report, which may include numeric data or model performance
+visualizations. Let's look at the typical sequence (after
+[configuration](/doc/config)):
+
+∞ `cml runner` launches a runner hosted by a cloud compute provider or
+[on-premise](/doc/self-hosted-runners).
+
+∞ `cml pr` commits a set of files to a new branch and create a pull request.
+
+∞ `cml comment` posts a Markdown report as a comment on a commit or pull/merge
+request.
```

Comment:
I prefer the list of commands or a table as suggested in https://github.com/iterative/cml.dev/pull/382#discussion_r1032707761 but up to you @casperdcl, feel free to remove it with:

```suggestion
into a CML report, which may include numeric data or model performance
visualizations. Select any command from the navigation bar to learn more.
```

Reply:
Were there any other concerns to get this merged?